patternfly/patternfly-elements
# feat(button): expose `button` part

## What I did

1. I added a new CSS variable to override the positioning of the `block` variant. I'm using a list of `pf-button`s in [`pf-pagination`](https://deploy-preview-2455--patternfly-elements.netlify.app/components/pagination/demo/), under the per-page menu, and I need to set the justify-content property to `start` instead of `center`. 

## Testing Instructions

1. Verify that `center` is the default position on the `block` variant.
2. Set `--pf-c-button--block--JustifyContent` to something other than center and verify the change reflects correctly.

## Notes to Reviewers

1. Let me know if the variable name I chose works. I followed Patternfly's naming convention, but I wasn't sure if there was a separate one I should follow for variables that we define in our project.
